compaction/compaction.cc

```diff
@@ -1707,15 +1755,18 @@ static future<compaction_result> scrub_sstables_validate_mode(sstables::compacti
 
     clogger.info("Finished scrubbing in validate mode {} - sstable(s) are {}", sstables_list_msg, valid ? "valid" : "invalid");
 
+    uint64_t validation_errors = 0;
     if (!valid) {
         for (auto& sst : *sstables->all()) {
             co_await sst->move_to_quarantine();
+            ++validation_errors;
         }
     }
 
     co_return compaction_result {
         .new_sstables = {},
         .ended_at = db_clock::now(),
+        .validation_errors = validation_errors,
```

Comment:
Shouldn't we take this from `cts` above?

Reply:
I don't understand. Which `cts` do you mean?

Reply:
`auto cts = seastar::make_lw_shared<compaction_manager::task::compaction_task_stats>();`

Reply:
It's beyond the scope of that function though.